cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -616,6 +619,276 @@ async def _get_trustworthiness_score_async(
+
+    TLMHybrid should be used if you want to use a better model to generate responses but want to get cheaper and quicker trustworthiness score
+    evaluations by using smaller models.
+
+    ** The TLMHybrid object is not meant to be constructed directly.** Instead, use the [`Studio.TLMHyrid()`](../studio/#method-tlmhyrid)
+    method to configure and instantiate a TLMHybrid object.
+    After you've instantiated the TLMHybrid object using [`Studio.TLMHybrid()`](../studio/#method-tlmhyrid), you can use the instance methods documented on this page.
+    Possible arguments for `Studio.TLMHybrid()` are documented below.
+
+    Most of the input arguments for this class are similar to those for TLM, major differences will be described below.
+
+    Args:
+        response_model (str): LLM used to produce the response to the given prompt.
+            Do not specify the model to use for obtaining the trustworthiness score here, instead specify it in the options argument.
+            Models currently supported include "gpt-3.5-turbo-16k", "gpt-4", "gpt-4o", "claude-3-haiku".
+
+        quality_preset (TLMQualityPreset, default = "medium"): preset configuration to control the quality of TLM trustworthiness scores vs. runtimes/costs.
+            Note that this preset only applies to the model computing the trustworthiness score.
+            Supported options include: "medium", "low".
+
+        options (TLMOptions, optional): a typed dict of advanced configuration options.
+            Note that most of these options only apply to the model computing the trustworthiness score, except for "max_tokens", which applies to both models.
+            Available options (keys in this dict) include "model", "max_tokens", "num_consistency_samples", "use_self_reflection".
+            For more details about the options, see the documentation for [TLMOptions](#class-tlmoptions).
+
+        timeout (float, optional): timeout (in seconds) to apply to each TLM prompt.
+
+        verbose (bool, optional): whether to print outputs during execution, i.e., whether to show a progress bar when TLM is prompted with batches of data.
+    """
+
+    def __init__(
+        self,
+        api_key: str,
+        response_model: str,
+        quality_preset: TLMQualityPreset,
+        *,
+        options: Optional[TLMOptions] = None,
+        timeout: Optional[float] = None,
+        verbose: Optional[bool] = None,
+    ) -> None:
+        """
+        Use `Studio.TLMHybrid()` instead of this method to initialize a TLMHybrid object.
+        lazydocs: ignore
+        """
+        self._api_key = api_key
+        self._response_model = response_model
+
+        if quality_preset in {"high", "best"}:
+            raise ValidationError(
+                f"Invalid quality preset: {quality_preset}. TLMHybrid only supports 'low' and 'medium' presets."
+            )
```

Comment:
The quality preset here specifies the preset used in the `get_trustworthiness_score()` call, for which `medium`/`high`/`best` presets do the same thing (the difference between `high` and `best` is the number of candidate responses which is only relevant for `prompt()`).
So instead of offering options which don't actually change the quality of the score at all (it will be confusing(, we are just disallowing it.